pyvista/plotting/camera.py

```diff
@@ -31,6 +31,38 @@ def __init__(self):
         self._elevation = 0.0
         self._azimuth = 0.0
 
+    def __repr__(self):
+       """Print a repr specifying the id of the camera and its camera type."""
+       return (f'<{self.__class__.__name__} at {hex(id(self))}>')
+
+    def __eq__(self, other):
+        """Compare whether the relevant attributes of two camera are equal."""
```

Comment:
Sorry, forgot to add this in the review.
```suggestion
        """Compare whether the relevant attributes of two cameras are equal."""
```